crates/rpc/rpc-types/src/eth/call.rs

```diff
@@ -23,6 +23,17 @@ pub struct StateContext {
     pub transaction_index: Option<TransactionIndex>,
 }
 
+/// CallResponse for eth_callMany
+#[derive(Debug, Clone, Default, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct EthCallResponse {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    /// eth_call output (if no error)
+    pub output: Option<Bytes>,
+    /// eth_call error output
+    pub error: Option<Bytes>,
```

Comment:
this should match `eth_call`?

is the error field also supposed to represent the error output of `eth_call`? if so then this should be a String?

Reply:
Ah okay! I can always do a `err.to_string()`

One question I do have is how does the RPC API handle the conversion from `Error` to `String`?  E.g. specifically for the `eth_call` method, the output is `Result<Bytes, EthApiError>`, where and when does the program know to convert the `EthApiError` to a `String`?

Reply:
like this: 

https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc/src/eth/error.rs#L537-L547

which wraps the output in an error which formats it on `.to_string`

Reply:
Where does the error get formatted into a string? or is that handled by serde?